core/primitives/src/types.rs

```diff
@@ -19,8 +19,12 @@ pub type AccountId = CryptoHash;
 pub type StructSignature = Signature;
 /// Hash used by a struct implementing the Merkle tree.
 pub type MerkleHash = CryptoHash;
-/// Part of the BLS signature.
-pub type BLSSignature = Signature;
+/// Mask which authorities participated in multi sign.
+pub type AuthorityMask = Vec<bool>;
+/// Part of the signature.
+pub type PartialSignature = Signature;
```

Comment:
I think name `BLSSignature` is clearer than `MultiSignature` which is less specific. Especially since we might have two types of multisignatures in our infra.

Reply:
well, the whole point of having a type alias, is that we may switch how we are doing the multi sign.
Like right now it's not BLS, but just a vec of signatures.